web/tests/stream_data.test.cjs

```diff
@@ -249,6 +261,14 @@ test("basics", () => {
     const stream_starting_with_25 = {
         name: "25-or-6-to-4",
         stream_id: 400,
+        can_add_subscribers_group: admins_group.id,
+        can_administer_channel_group: admins_group.id,
+        can_subscribe_group: admins_group.id,
+        subscribed: false,
+        is_muted: false,
+        invite_only: false,
+        history_public_to_subscribers: true,
+        is_web_public: true,
```

Comment:
`is_web_public` can be set to `false` here itself, so that we need not change it below.